tests/containers/containerd_rmt.pm

```diff
@@ -29,18 +29,17 @@ sub run {
     install_kubectl();
     install_helm();
 
-    my $rmt_helm = get_var('RMTTEST_CHART', 'https://github.com/SUSE/helm-charts/archive/refs/heads/main.tar.gz');
+    my $rmt_helm = get_var('RMTTEST_REPO', 'https://github.com/SUSE/helm-charts/archive/refs/heads/main.tar.gz');
     # pull in the testsuite
     assert_script_run("curl -sSLk $rmt_helm | tar -zxf -");
     my $rmt_config = get_var('RMTTEST_CONFIG', 'https://gitlab.suse.de/QA-APAC-I/testing/-/raw/master/data/rmtcontainer/myvalue.yaml');
     assert_script_run("curl -sSLOk $rmt_config");
-    my $rmt_repo = get_required_var('RMTTEST_REPO');
-    my $rmt_tag = get_var('RMTTEST_TAG', 'latest');
-    assert_script_run("helm install --set app.image.repository=$rmt_repo --set app.image.tag=$rmt_tag rmt ./helm-charts-main/rmt-helm/ -f myvalue.yaml");
+    assert_script_run("helm install rmt ./helm-charts-main/rmt-helm -f myvalue.yaml");
```

Comment:
How is now the to-test image being passed to the helm chart?

Reply:
I'm debugging why it's failing with the image.